mmv1/products/logging/LogView.yaml

```diff
@@ -27,17 +27,31 @@ references: !ruby/object:Api::Resource::ReferenceLinks
 # Skipping sweeper since this is a child resource.
 skip_sweeper: true
 description: 'Describes a view over log entries in a bucket.'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  skip_import_test: true
```

Comment:
I know you were testing some things, so I think it might be worth circling back to see if tests still pass without some of these settings.

For `skip_import_test`, do you need this?

Reply:
Done.

Reply:
Per the field specification, this field is `Used to handle situations where typical generated IAM tests cannot import due to the parent resource having an API-generated id`. I could be wrong, but I don't think that applies here, and we may be skipping tests simply because they aren't configured properly.

You may be able to fix the error you saw by tweaking the `import_format` and/or the `primary_resource_name`. My guess would be that since you are allowing imports by `{{name}}`, you could probably change `primary_resource_name` to simply `'fmt.Sprintf("tf-test-my-view%s", context["random_suffix"])'`.

Let me know if you run into trouble, or feel that this does in fact fit the use-case for `skip_import_test`.

Reply:
First I admit I am not why change `primary_resource_name` under " - !ruby/object:Provider::Terraform::Examples" will affect generated tests.

But I still tried `primary_resource_name: 'fmt.Sprintf("tf-test-my-view%s", context["random_suffix"])'`
Then I got 
```
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google-beta/services/logging -v -run=TestAccLoggingLogViewIamPolicyGenerated_withCondition -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google-beta/version.ProviderVersion=acc"
# github.com/hashicorp/terraform-provider-google-beta/google-beta/services/logging_test
# [github.com/hashicorp/terraform-provider-google-beta/google-beta/services/logging_test]
google-beta/services/logging/iam_logging_log_view_generated_test.go:56:24: fmt.Sprintf format %s reads arg #3, but call has 2 args
...
```

Then I looked at the generated code, it shows:
```
ImportStateId:     fmt.Sprintf("%s/locations/%s/buckets/%s/views/%s roles/logging.admin", envvar.GetTestRegionFromEnv(), fmt.Sprintf("tf-test-my-view%s", context["random_suffix"])),
```